.github/workflows/ci.yml

```
@@ -389,7 +389,7 @@ jobs:
           brew install diffutils
 
       - name: Test
-        timeout-minutes: 20
+        timeout-minutes: 30
```

Linux test has a timeout set to 25 minutes (for `huge, clang, asan, 5.1` that run in 22 minutes).
Changing the macos test timeout to 30 seems too redundant, so I'll set it to 25 minutes.

```suggestion
        timeout-minutes: 25
```